### PR TITLE
CHARMS RAS @CP2-3730 @CP2-3703 Verifying participants can complete screeners, verifying that the Consent Record field Assent signed is true after minor aged 11-13 submits assent.

### PR DESCRIPTION
**Pull request by katm007**
Verifying participants can complete screeners + verifying that the Consent Record field Assent signed is true after minor aged 11-13 submits assent.

### CODE REVIEW

File: src/test/java/ServiceNow/CHARMS/Steps/RAS_All_Steps.java [hunk 1]
```diff
@@ -286,25 +283,18 @@ public static void PI_completes_consent_and_verifies_in_Native_View(String sheet
      * @param pdfName                   The name of the PDF file to be downloaded.
      */
     @Given("{string} text shows on participant portal and when clicked downloads {string}")
-    public static void text_shows_on_participant_portal_and_when_clicked_downloads_consent_form(String expectedDownloadStudyText, String pdfName) {
+    public static void text_shows_on_participant_portal_and_when_clicked_downloads_consent_form(String expectedDownloadStudyText, String pdfName) throws IOException {
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/CHARMS/Steps/RAS_All_Steps.java:286]: Any reason for throwing IOException? It does not seem to be used here.

File: src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RAS_Screener_Submissions_StepsImpl.java [hunk 2]
```diff
@@ -73,6 +73,18 @@ public void rasScreenerScenarioSelector(String sheetName) {
         } else if (sheetName.contentEquals("screenerScenario5")) {
             ras_Screener_TestDataManager.dataInitializerRasScreener(sheetName);
 //            ras_scenario_five_stepsImpl.ras_screener_submission_five();
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RAS_Screener_Submissions_StepsImpl.java:75]: Commented line 75 can be removed if no longer in use

**Review by hamidnesar — COMMENTED** [review]
Hi, please review the comments. thanks!

File: src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RAS Entire Flow.feature [hunk 3]
```diff
@@ -7,20 +7,20 @@ Feature: RAS Screener Scenarios
     Given a participant is on the RASopathies Longitudinal Cohort Study login page "myRASLoginPage"
     And logs in via Okta with username "<Email>" and password "<Password>"
     And clicks on "Eligibility Questionnaire" to begin survey
-#    When the participant submits a screener from excel sheet "<ScreenerScenario>"
+    When the participant submits a screener from excel sheet "<ScreenerScenario>"
 #    Then screener data from "<ScreenerScenario>" is verified
```
[the remark this answers is not in the document]
Comment (katm007) [reply src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/RAS Entire Flow.feature:11]: Yes, that's a task I will eventually do after I take care of higher priority items.

File: src/test/java/ServiceNow/CHARMS/Steps/RAS_All_Steps.java (hunk 1 above)
Comment (katm007) [reply src/test/java/ServiceNow/CHARMS/Steps/RAS_All_Steps.java:286]: Fixed!

File: src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RAS_Screener_Submissions_StepsImpl.java (hunk 2 above)
Comment (katm007) [reply src/test/java/ServiceNow/CHARMS/StepsImplementation/RASStudy/RAS_Screener_Submissions_StepsImpl.java:75]: This is something Diego was working on so I can't remove it.

**Review by hamidnesar — APPROVED** [review]
looks good!

**Review by alenaminsk — APPROVED** [review]
Approved

**Pull request merged**